client/src/User/UserHome/UserHome.js

```diff
@@ -57,7 +57,7 @@ class UserHome extends Component {
 
   render() {
     return (
-      <div style={{ padding: "50px 200px" }}>
+      <div style={{ margin: "5%" }}>
```

Comment:
Why did you change the styling to margin 5%? The original styling with padding: "50px 200px"make the layout look even throughout our app.

Reply:
![Example](https://user-images.githubusercontent.com/21010314/48996797-670b3980-f11b-11e8-9344-b282d461d269.PNG)
If you take a look at the website currently in production and resize your window, the padding will remain fixated on padding the div by 200 pixels. Instead of having static styling, it's better to change the entire User Interface based on percentages or viewport which makes the entire website responsive. When we make our website responsive, it will also solve our issue in displaying our website on the mobile device.

Reply:
Oh yes, that make sense. Thanks for clarifying this up!